android/faq_android_studio.md

```diff
@@ -0,0 +1,73 @@
+# Android Studio FAQ
+
+## Build Issues
+
+_What do I do if building in Android Studio fails (and the same version is building for everyone else)?_
+
+_Try the following in this order:_
+
+1. Verify the desired Build Variant is selected
+2. Clean project then rebuild
+3. Sync project with Gradle files
+4. Invalidate Caches and Restart
+5. Update Android Studio and Restart
+6. Delete the Gen directory and rebuild
+7. Wipe data on emulator
+8. Delete the ```~/.gradle``` directory (see: Build Issue - Prevention below)
+9. Wipe Android Studio cache ```rm -rf ~/.idea```
+10. Wipe data on emulator
+
+### Build Issue - Prevention
+Mike’s Daily Routine for Build Issue Prevention
```

Comment:
This is specifically designed to work around the problems with SNAPSHOTs: I don't really want to land this in shared-docs because 1) most projects don't use snapshots and 2) when we remove snapshots, this isn't expected to be a problem.